errors/errctx.go

```diff
@@ -28,8 +28,8 @@ func Newf(ctx context.Context, format string, args ...interface{}) error {
 	return ErrCtx{ctx: ctx, err: errgo.Newf(format, args...)}
 }
 
-func NoteMask(ctx context.Context, err error, message string) error {
-	return ErrCtx{ctx: ctx, err: errgo.NoteMask(err, message)}
+func NoteMask(ctx context.Context, err error, message string, pass ...func(error) bool) error {
+	return ErrCtx{ctx: ctx, err: errgo.NoteMask(err, message, pass...)}
 }
```

Comment:
Sorry I didn't know about this function but it should not be implemented. We don't use and `*Mask*` function since a loooong time. Any call to a Mask function should be converted to a `Notef` call.
```suggestion
```